### ISSUE TITLE
Sortings sessions table by  "Is Occurrence?" results in SQL error

### ISSUE THREAD

**Issue by mihob** [verbb/Events#161]
### Describe the bug

Sorting sessions table by "Is Occurenrce?" results in SQL error

<img width="1059" alt="grafik" src="https://github.com/user-attachments/assets/4e327920-d5d4-443d-81a4-315534be89c5">

SQL error:

```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'recurring' in 'order clause' The SQL being executed was: SELECT `elements`.`id`, `elements`.`canonicalId`, `elements`.`fieldLayoutId`, `elements`.`uid`, `elements`.`enabled`, `elements`.`archived`, `elements`.`dateLastMerged`, `elements`.`dateCreated`, `elements`.`dateUpdated`, `elements_sites`.`id` AS `siteSettingsId`, `elements_sites`.`siteId`, `elements_sites`.`title`, `elements_sites`.`slug`, `elements_sites`.`uri`, `elements_sites`.`content`, `elements_sites`.`enabled` AS `enabledForSite`, `events_sessions`.`id`, `events_sessions`.`primaryOwnerId`, `events_sessions`.`startDate`, `events_sessions`.`endDate`, `events_sessions`.`allDay`, `events_sessions`.`groupUid`, `events_elements_sites`.`slug` AS `eventSlug`, `events_event_types`.`handle` AS `eventTypeHandle`, `elements_owners`.`ownerId`, `elements_owners`.`sortOrder` FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `siteSettingsId` FROM `elements` `elements` INNER JOIN `events_sessions` `events_sessions` ON `events_sessions`.`id` = `elements`.`id` INNER JOIN `elements_owners` `elements_owners` ON (`elements_owners`.`elementId` = `elements`.`id`) AND (`elements_owners`.`ownerId`=1987) LEFT JOIN `events_events` `events_events` ON `elements_owners`.`ownerId` = `events_events`.`id` LEFT JOIN `events_event_types` `events_event_types` ON `events_events`.`typeId` = `events_event_types`.`id` INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id` WHERE (`elements_sites`.`siteId`=1) AND (`elements`.`archived`=FALSE) AND (`elements`.`dateDeleted` IS NULL) AND (`elements`.`draftId` IS NULL) AND (`elements`.`revisionId` IS NULL) ORDER BY `recurring` LIMIT 50) `subquery` INNER JOIN `elements` `elements` ON `elements`.`id` = `subquery`.`elementsId` INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`siteSettingsId` INNER JOIN `events_sessions` `events_sessions` ON `events_sessions`.`id` = `subquery`.`elementsId` INNER JOIN `elements_owners` `elements_owners` ON (`elements_owners`.`elementId` = `elements`.`id`) AND (`elements_owners`.`ownerId`=1987) LEFT JOIN `events_events` `events_events` ON `elements_owners`.`ownerId` = `events_events`.`id` LEFT JOIN `events_event_types` `events_event_types` ON `events_events`.`typeId` = `events_event_types`.`id` LEFT JOIN `elements_sites` `events_elements_sites` ON `elements_owners`.`ownerId` = `events_elements_sites`.`elementId` and `events_elements_sites`.`siteId` = `elements_sites`.`siteId` ORDER BY `recurring`
```

### Steps to reproduce

1.  Try sorting the session table by “Is Occurrence”

### Craft CMS version

5.5.0.1

### Plugin version

3.0.1

### Multi-site?

YES

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/events:"dev-craft-5 as 3.0.2"`.

**Comment by engram-design** [comment]
Fixed in [3.0.3](https://github.com/verbb/events/releases/tag/3.0.3)
